olive/resource_path.py

```diff
@@ -329,21 +325,93 @@ def save_to_dir(self, dir_path: Union[Path, str], name: str = None, overwrite: b
         logger.debug(f"Downloading model {self.config.name} version {self.config.version} to {new_path}.")
         from azure.core.exceptions import ServiceResponseError
 
+        max_tries = self.config.azureml_client.max_operation_retries if self.config.azureml_client else 3
```

Comment:
I don't think this line is needed since both children classes provide the value?

Reply:
Oh I forgot to remove this. Thanks!